scripts/analyze_csv_data.py

```diff
@@ -681,9 +684,23 @@ def gen_all_improvement_plots(outputs, prefix):
 # bisco scaling vs no scaling (NO JOIN DEFAULT)
 df = pd.read_csv(os.path.join(CSV_DIR, 'miplib_scaling_100k.csv'))
 df = fill_in_missing_problems(df, miplib_instances)
+
+# Pull out best performing scaling for each instance / tolerance:
```

Comment:
Pre-existing: I believe the comment at line 605 should be "Pull out best step-size rule for each instance...".

Reply:
Fixed, thanks.